From 2dec6bca2bcea775804e02681244130c124861bc Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Wed, 14 Mar 2012 12:22:27 +0530
Subject: [PATCH 0911/1092] ARM: OMAP3+: am33xx: Fix pruss clk alias name

Correct the clock alias name for pruss clock, which was
created using old clock tree data, and which is no longer valid.

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 0d5fa7b..2645cdf 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -2336,10 +2336,8 @@ static void __init am335x_evm_init(void)
 	omap_board_config = am335x_evm_config;
 	omap_board_config_size = ARRAY_SIZE(am335x_evm_config);
 	/* Create an alias for icss clock */
-	if (clk_add_alias("pruss", NULL, "icss_uart_gclk", NULL))
+	if (clk_add_alias("pruss", NULL, "pruss_uart_gclk", NULL))
 		pr_warn("failed to create an alias: icss_uart_gclk --> pruss\n");
-	if (clk_add_alias("pruss", NULL, "icss_fck", NULL))
-		pr_warn("failed to create an alias: icss_fck --> pruss\n");
 	/* Create an alias for gfx/sgx clock */
 	if (clk_add_alias("sgx_ck", NULL, "gfx_fclk", NULL))
 		pr_warn("failed to create an alias: gfx_fclk --> sgx_ck\n");
-- 
1.7.11.2

